html-skeleton/index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeletons</title>
+    <body>Hello, HTML!</body></html>
```

Comment:
You are missing your closing tag for the `head` element. Also, please move the closing tag for the `html` element onto its own line.